plugins/woocommerce-blocks/assets/js/blocks/product-elements/add-to-cart-form/settings.tsx

```diff
@@ -42,36 +52,66 @@ const getHelpText = ( quantitySelectorStyle: QuantitySelectorStyle ) => {
 	}
 };
 
-export const Settings = ( {
+export const AddToCartSettings = ( {
```

Comment:
I would name this component `AddToCartFormSettings`, to keep it consistent with `AddToCartFormEdit`. What do you think?